README.md

```diff
@@ -174,7 +174,7 @@ The list of available tasks for managing the chain is as follows:
 - `chain-start` - Run the full node okp4d application using the chain's home directory under a Docker container.
 - `chain-stop` - Stop the chain's container.
 
-### Initialize the chain
+## 🚀 Initialize the chain
```

Comment:
I'm wondering if this section as well as `### 🟢 Start the chain`, `### 🛳 Deploy the Smart Contracts`, `### 🎮 Free execution of the CLI command`,`### 🔴 Stop the chain` and `### 🧼 Clean the chain`, should be placed in the `💻 Develop` section ?

Reply:
Done :heavy_check_mark: